pages/validator/external-chains/binance.md

```diff
@@ -1,149 +1,270 @@
+cd $HOME
+mkdir binaries && cd binaries
+
+# if you are on linux amd
+wget https://github.com/bnb-chain/bsc/releases/download/$BSC_GETH_RELEASE/geth_linux
+mv geth_linux geth
+chmod +x *
+sudo mv * /usr/bin/
+# verify version
+geth version
+cd $HOME
 ```
-wget https://github.com/bnb-chain/bsc/releases/latest/download/mainnet.zip
+
+
+
+<Tabs tabs={[
+{
+title: "Mainnet",
+content: <CodeBlock language="bash">
+{`mkdir -p $HOME/.bsc/config
+cd $HOME/.bsc/config
+wget https://github.com/bnb-chain/bsc/releases/download/$BSC_GETH_RELEASE/mainnet.zip
 unzip mainnet.zip
-```
-```
-mv geth_linux /usr/bin/geth
-chmod +x /usr/bin/geth
-rm -rf testnet.zip
-```
-## 4. Configure config.toml file
-```
-nano config.toml
-```
-Inside the ``config.toml`` found line ``HTTPHost = "127.0.0.1"`` and change IP from ``127.0.0.1`` on your IP address.
+rm mainnet.zip
+`}
+</CodeBlock>
+},
+{ title: "Testnet", content: <CodeBlock language="bash">
+{`mkdir $HOME/.bsc/config
+cd $HOME/.bsc/config
+wget https://github.com/binance-chain/bsc/releases/download/$BSC_GETH_RELEASE/testnet.zip
+unzip testnet.zip
+rm testnet.zip
+`}
+</CodeBlock>
+}
+]} />
 
-Scroll down and delete following lines:
-```
-[Node.LogConfig]
-FileRoot = ""
-FilePath = "bsc.log"
-MaxBytesSize = 10485760
-Level = "info"
-```
-Save it by command ``CTRL+X,Y``
 
-## 5. Write state genesis localy
-```
-geth --datadir node init genesis.json
+
+
+## Initialize the node or sync from snapshot
+
+#### 1. If you want to sync from genesis
+```bash
+cd $HOME
+geth --datadir $HOME/.bsc init genesis.json
 ```
-## 6. Configure systemd service file
+#### 2. If you want to sync from snapshot - Mainnet Only
+You find the snapshot links  [here](https://github.com/bnb-chain/bsc-snapshots) or [here](https://github.com/48Club/bsc-snapshots#download) to sync from snapshot
+
+```bash
+# change the url to latest snapshot available: you can find it in the links shared above
+GETH_SNAPSHOT_URL=https://snapshots.48.club/geth.24761776.tar.lz4
+```
+```bash
+cd $HOME/.bsc/
+aria2c -s14 -x14 -k100M $GETH_SNAPSHOT_URL -o geth.tar.lz4
+# check checksum: it will take time to calculate checksum
+openssl sha256 geth.tar.lz4
+# uncompress the file
+lz4 -cd geth.tar.lz4 | tar xf -
+rm geth.tar.lz4
 ```
-tee /etc/systemd/system/bscd.service > /dev/null <<EOF
+
+## Create Service
+```bash
+sudo tee /etc/systemd/system/geth.service > /dev/null <<EOF
 [Unit]
-Description=BSC
-After=network-online.target
+Description=Geth node
+After=online.target
+
 [Service]
-User=root
-ExecStart=/usr/bin/geth --config root/bsc/config.toml --datadir root/bsc/node --ws --ws.origins '*'
-Restart=always
+Type=simple
+User=root 
+ExecStart=/usr/bin/geth --config $HOME/.bsc/config/config.toml --txlookuplimit=0 --syncmode=full --tries-verify-mode=none --pruneancient=true --diffblock=5000 --cache 8000 --rpc.allow-unprotected-txs --datadir $HOME/.bsc --http --http.vhosts "*" --http.addr 0.0.0.0 --ws --ws.origins '*' --ws.addr 0.0.0.0 --http.port 8545
```

Comment:
We should tell user to apply the appropriate firewall rules from infra provider or OS level if they want to keep the node private, otherwise `--http.addr 0.0.0.0` and `--ws.addr 0.0.0.0` will publicly exposes it.

Reply:
Added a note